pkg/query/assetsapi.go

```diff
@@ -0,0 +1,104 @@
+package query
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/kubecost/cost-model/pkg/kubecost"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type assetResponse struct {
+	Code int                    `json:"code"`
+	Data []map[string]AssetNode `json:"data"`
+}
+
+type AssetParameters struct {
+	RestConfig *rest.Config
+	Ctx        context.Context
+
+	KubecostNamespace  string
+	ServiceName        string
+	Window             string
+	Aggregate          string
+	DisableAdjustments bool
+	Accumulate         bool
+	UseProxy           bool
+	FilterTypes        string
+}
+
+// QueryAssets queries /model/assets by proxying a request to Kubecost
+// through the Kubernetes API server if useProxy is true or, if it isn't, by
+// temporarily port forwarding to a Kubecost pod.
+func QueryAssets(p AssetParameters) ([]map[string]AssetNode, error) {
+
+	// aggregate, accumulate, and disableAdjustments are hardcoded;
+	// as other asset types are added in to be filtered by, this may change,
+	// but for now anything beyond isn't needed.
+
+	requestParams := map[string]string{
+		"window":             p.Window,
+		"aggregate":          "",
+		"accumulate":         "true",
+		"disableAdjustments": "false",
```

Comment:
For `aggregate`, `accumulate`, and `disableAdjustments`, either use the value from the parameter struct or remove the option from the parameter struct entirely. I would recommend removing these from the parameter struct entirely as I don't think we have a use case yet for configuring these and I don't anticipate one soon.